Add slug field description (#223)
mirumee/saleor-docs: docs/developer/slug-fields.mdx

---
title: Slug fields
---

## Introduction

> Slug is a short label for an object. Main reason for using slugs is ability to encorporate them in URLs. Translated names can contain all utf-8 signs, which could lead to URLs which could not be written for people with different keyboard layout.

Reviewer: A slug is a short label that can explain an object or page's content. The main reason for using a slug is the ability to incorporate it to URLs. Translated names can contain all UTF-8 signs—this allow for URLs to be written by people with different keyboard layouts.

> Slug is a short label for an object. Main reason for using slugs is ability to encorporate them in URLs. Translated names can contain all utf-8 signs, which could lead to URLs which could not be written for people with different keyboard layout.

Reviewer: ```suggestion
A slug is a short label that can explain an object or page's content. The main reason for using a slug is the ability to incorporate it to URLs. Translated names can contain all UTF-8 signs—this allow for URLs to be written by people with different keyboard layouts.
```

Slug can contain:
- letters
- numbers
- underscores
- hyphens

> By convention slugs in Saleor are not translatable, so the users using different languages could use the same product URLs.

Reviewer: Slugs are not translatable in Saleor. When using different languages users should apply the same product URLs regardless of the language.

> By convention slugs in Saleor are not translatable, so the users using different languages could use the same product URLs.

Reviewer: ```suggestion
Slugs are not translatable in Saleor. When using different languages users should apply the same product URLs regardless of the language.
```

## Example

> As example we would like to create product called "Żółta ćma" (Yellow moth). Expected slug for such product would be `Zolta-cma`.

Reviewer: For example: we would like to create a product called "Żółta ćma" (Yellow moth). The expected slug for such product would be `Zolta-cma`.

> As example we would like to create product called "Żółta ćma" (Yellow moth). Expected slug for such product would be `Zolta-cma`.

Reviewer: ```suggestion
For example: we would like to create a product called "Żółta ćma" (Yellow moth). The expected slug for such product would be `Zolta-cma`.
```
